library/Zend/Console/Prompt/Checkbox.php

```diff
@@ -0,0 +1,178 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+namespace Zend\Console\Prompt;
+
+use Zend\Console\Exception;
+
+class Checkbox extends Char
```

Comment:
Do you actually need to inherit from `Char`? Seems like it is not really a subtype (from a logical perspective)